apps/files/src/components/FilesApp.vue

```diff
@@ -1,24 +1,21 @@
   <template>
-    <v-content>
-      <v-container id="files-app" fluid pa-0>
-      <v-layout row fill-height>
-        <v-flex :class="{'xs12': selectedFiles.length === 0, 'xs6': selectedFiles.length > 0 }" pa-0 fill-height>
+    <div class="oc-app" id="files-app">
+      <oc-app-content>
```

Comment:
I'd expect oc-app-content to be the root element here

Reply:
We got some problems here, in this approach oc-app-content and oc-app-sidebar both are child components to oc-app which for now is a css-class and needs to be adopted

Reply:
> Same as in the first comment - The fixed slots give as the chance to controll app styles to be within phoenix design.

from my understanding the common design comes from the individual properly designed components and not by a globally defined stylesheet which creates a mess of css classes.

In this case we could have stayed with the original uikit apporach ....

Reply:
> In this case we could have stayed with the original uikit apporach ....

I just tried to keep things newly build as far away from vuetify as possible to give us the chance to eventually migrate to a own pattern library easier - which was not part of the discussion as vuetify came into play